### PR TITLE
Add tetragon v1.3.0@4a6643e71e2357324ab8ab73ecb7286d73e96ef8 ⎈

### PR DESCRIPTION
**Pull request by kkourt**

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -20945,9 +20955,9 @@ entries:
     version: 1.2.1
   - apiVersion: v2
     appVersion: 1.2.0
-    created: "2024-09-05T13:21:20.070471945Z"
+    created: "2024-12-13T13:21:53.111318705Z"
     description: Helm chart for Tetragon
-    digest: 711faa2847966e52323f92e50ca5da69a462046470a2427176d273b1e1899d8e
+    digest: c3bfb7cebf8fd777120c07ea1b6707436e6219fdde4bd9966f55bfbc79a5c295
```
Comment (qmonnet) [inline index.yaml:20960]: Why are these modified?

**Review by qmonnet — COMMENTED** [review]
I'm a bit curious about why there were so many duplicate entries in the README.md, now removed by your PR. Not curious enough to dig, though :).

One question below

File: index.yaml (hunk 1 above)
Comment (kkourt) [reply index.yaml:20960]: No idea 🤷🏼 

The commit is automatically generated from a script (`./generate_helm_release.sh`)

Comment (qmonnet) [reply index.yaml:20960]: Yes... OK let's assume the script works, I guess you'll see soon enough if someone complains :) :+1:

Comment (kkourt) [reply index.yaml:20960]: And the checksum seems wrong 🤔 

```
sha256sum tetragon-1.2.0.tgz tetragon-1.3.0.tgz 
711faa2847966e52323f92e50ca5da69a462046470a2427176d273b1e1899d8e  tetragon-1.2.0.tgz
19bb9f3777ec823d43e7e5a1d2d6c8ef2320b54372f493d3fe460a0f842995e3  tetragon-1.3.0.tgz
```
